pkg/apis/infinispan/v1/infinispan_types.go

```diff
@@ -249,6 +249,31 @@ type InfinispanExternalDependencies struct {
 	// Name of the persistent volume claim with custom libraries
 	// +optional
 	VolumeClaimName string `json:"volumeClaimName,omitempty"`
+	// +optional
+	Artifacts []InfinispanExternalArtifacts `json:"artifacts,omitempty"`
```

Comment:
@ryanemerson , @rigazilla what do you think about to add a downloading type strategy (stop | ignore) and give a user to decide what to do in case of fails download in any case (hash validation or incorrect or unavailable location)

Reply:
> @ryanemerson , @rigazilla what do you think about to add a downloading type strategy (stop | ignore) and give a user to decide what to do in case of fails download in any case (hash validation or incorrect or unavailable location)

IMO starting a cluster with a missing piece is not a common use case. I would keep things as they are and maybe document the fact that CR needs to be deleted and recreated, if usability can't be improved.

Reply:
Good point, agree. In this case I would prefer to keep this PR on hold and implement #1076 before. I will to take about it.
Than, I will plan to thinking about OCP/Kaniko build integration.